web/src/components/storage/ProposalVolumes.jsx

```diff
@@ -56,6 +57,29 @@ const sizeText = (size) => {
   return filesize(size, { base: 2 });
 };
 
+const AutoCalculatedTooltip = ({ volume, children }) => {
+  // the size is not affected by snapshots or other volumes
+  if (!volume.snapshotsAffectSizes && volume.sizeRelevantVolumes.length === 0) {
+    const content = <Text>These limits are not affected by any other settings</Text>;
+    return <Tooltip content={content}>{children}</Tooltip>;
+  }
```

Comment:
If it's not affected then there is no tooltip because there is no "auto-calculated" label. Isn't it?

Reply:
Yes, that's it.